<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Preferences</class>
 <widget class="QWidget" name="Preferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>508</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="textLabel1_6">
       <property name="toolTip">
        <string>Path where the binary files are created.</string>
       </property>
       <property name="whatsThis">
        <string>Several information (such as the description of the documents or the corresponding indexing files) are stored as binary files in a given path (a sub-directory is created for each session).</string>
       </property>
       <property name="text">
        <string>Storage Path</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="IndexDir"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Plug-ins Paths</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QListWidget" name="Dirs">
        <property name="toolTip">
         <string>Paths where to search the plug-ins</string>
        </property>
        <property name="whatsThis">
         <string>Specify the different directories where the application should look for plug-ins. It looks recursively. When these paths are changed, the application must be relaunched.</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="_77">
        <item>
         <widget class="QPushButton" name="buttonAdd">
          <property name="text">
           <string>Add ...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonDel">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>100</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="_5">
     <item>
      <widget class="QLabel" name="textLabel4_2">
       <property name="text">
        <string>Scripting Path</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="PrgPath"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
